src/components/Ads2.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Grid } from "semantic-ui-react";
+import lagavulinAd from "../images/lagavulinAd.jpg";
+
+const Ads2 = () => {
```

Comment:
it is unnecessary to have seperate components

reuse the same component

Reply:
We have one component now serving two different ads